<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>613</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>FRC 2429 Blockhead Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="qt_tree_widget_nt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Age</string>
     </property>
    </column>
   </widget>
   <widget class="QPlainTextEdit" name="qt_text_status">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>921</width>
      <height>51</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="qt_button_refresh_nt">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh network tables</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>181</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Intake/Indexer</string>
    </property>
    <widget class="QLabel" name="qlabel_intake_indicator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 7px;
border-radius: 7px;
background-color:rgb(80, 235, 0);</string>
     </property>
     <property name="text">
      <string>Intake On</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_intake_piston_indicator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 7px;
border-radius: 7px;
background-color:rgb(80, 235, 0);</string>
     </property>
     <property name="text">
      <string>Intake Extended</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_indexer_indicator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 7px;
border-radius: 7px;
background-color:rgb(80, 235, 0);</string>
     </property>
     <property name="text">
      <string>Indexer On</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>181</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Shooter</string>
    </property>
    <widget class="QLabel" name="qlabel_shooter_speed_indicator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 7px;
border-radius: 7px;
background-color:rgb(80, 235, 0);</string>
     </property>
     <property name="text">
      <string>Shooter 
 At Speed</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="qlcd_shooter_rpm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Filled</enum>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_shooter_indicator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 7px;
border-radius: 7px;
background-color:rgb(80, 235, 0);</string>
     </property>
     <property name="text">
      <string>Shooter On</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>132</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Shooter RPM</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>181</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Climber</string>
    </property>
    <widget class="QLabel" name="qlabel_climber_indicator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 7px;
border-radius: 7px;
background-color:rgb(80, 235, 0);</string>
     </property>
     <property name="text">
      <string>Climber On</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="qlcd_climber_current">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_long_arm_indicator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>226</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 7px;
border-radius: 7px;
background-color:rgb(80, 235, 0);</string>
     </property>
     <property name="text">
      <string>Long Arm</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_short_arm_indicator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 7px;
border-radius: 7px;
background-color:rgb(80, 235, 0);</string>
     </property>
     <property name="text">
      <string>Short  Arm</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>84</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Climber Current</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="qlabel_camera_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Camera View</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>181</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Pneumatics</string>
    </property>
    <widget class="QLabel" name="qlabel_compressor_indicator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 7px;
border-radius: 7px;
background-color:rgb(80, 235, 0);
color:rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Compressor On</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>qt_text_status</zorder>
   <zorder>qt_button_refresh_nt</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>qlabel_camera_view</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>qt_tree_widget_nt</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1096</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>NetworkTables</string>
    </property>
    <addaction name="qaction_show_hide"/>
    <addaction name="qaction_refresh"/>
   </widget>
   <widget class="QMenu" name="menuNetworkTables">
    <property name="title">
     <string>Camera</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuNetworkTables"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="qaction_show_hide">
   <property name="text">
    <string>Show / Hide</string>
   </property>
  </action>
  <action name="qaction_refresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
